python/doc/examples/numerical_methods/optimization/plot_ego.py

```diff
@@ -210,6 +210,7 @@
 # and we load the model function :
 model = bm.model
 objectiveFunction = model.getMarginal(0)
+noiseModel = model.getMarginal(1)
```

Comment:
For the sake of consistency, the Branin model should be updated:

## API doc ([python/src/usecases/branin_function.py](https://github.com/openturns/openturns/blob/ac0a66bef3efb6a1ac73620eb79cb2c186a44bf5/python/src/usecases/branin_function.py))

![image](https://user-images.githubusercontent.com/31989332/161000059-252ec084-d7d1-46b2-b2d2-f849f678d604.png)

**model** should only contain the Branin function, and a new **noiseModel** attribute should be created.

## Common use cases doc ([python/doc/usecases/use_case_branin.rst](https://github.com/openturns/openturns/blob/b060938db798457a0db5f0b2ac169771a0f8fa4c/python/doc/usecases/use_case_branin.rst))

![image](https://user-images.githubusercontent.com/31989332/161001061-06c90899-10d4-4aff-b00b-72d02a2e2d74.png)

BTW, this doc is quite unclear. It makes references to EGO without mentioning EGO. Perhaps the whole paragraph beginning with "This time the AEI formulation is used" should be deleted. What do you think @mbaudin47 @jschueller ?